### PR TITLE
Add Full Text Search Description to PGroonga

### PR DESCRIPTION
**Pull request by askdkc**
## What kind of change does this PR introduce?

PGroonga provides a Full Text Search function, which is a handy feature missing in the current documentation. Therefore, I have added this functionality to enhance the documentation.

## What is the current behavior?

Full Text Search function is missing in the doc.

## What is the new behavior?

Provides Full Text Search function description.

## Additional context

PGroonga provides a wide range of useful functions. While this is just the tip of the iceberg, it can be particularly helpful for new users getting started.

### CODE REVIEW

File: apps/docs/pages/guides/database/extensions/pgroonga.mdx [hunk 1]
```diff
@@ -100,6 +100,75 @@ The pgroonga index is used to retrive the result set:
 | 2   | 'Groonga is a fast full text search engine that supports all languages.' |
 ```
 
+## Full Text Search
+The &@~ symbol operator is for Full Text Search. It returns any matches result. Unlike `LIKE` operator, pgroonga can search any text that contains the keyword case insensitive.
```
Comment (dshukertjr) [inline apps/docs/pages/guides/database/extensions/pgroonga.mdx:104]:
```suggestion
The `&@~` operator performs full text search. It returns any maching results. Unlike `LIKE` operator, pgroonga can search any text that contains the keyword case insensitive.
```

File: apps/docs/pages/guides/database/extensions/pgroonga.mdx [hunk 2]
```diff
@@ -100,6 +100,75 @@ The pgroonga index is used to retrive the result set:
...
+ id |                                content                                 
+----+------------------------------------------------------------------------
+  2 | Groonga is a fast full text search engine that supports all languages.
+  3 | PGroonga is a PostgreSQL extension that uses Groonga as index.
+  4 | There is groonga command.
+(3 rows)
+```
+
+### Match all search words
+To find all memos where content contains BOTH of the words postgres and postgres, we can just use space to separate each words:
```
Comment (dshukertjr) [inline apps/docs/pages/guides/database/extensions/pgroonga.mdx:124]:
```suggestion
To find all memos where content contains BOTH of the words `postgres` and `pgroonga`, we can just use space to separate each words:
```

File: apps/docs/pages/guides/database/extensions/pgroonga.mdx [hunk 3]
```diff
@@ -100,6 +100,75 @@ The pgroonga index is used to retrive the result set:
...
+And the result:
+```markdown
+ id |                            content                             
+----+----------------------------------------------------------------
+  3 | PGroonga is a PostgreSQL extension that uses Groonga as index.
+(1 row)
+```
+
+### Match any search words
+To find all memos where content contain ANY of the words postgres or postgres, use the upper case OR:
```
Comment (dshukertjr) [inline apps/docs/pages/guides/database/extensions/pgroonga.mdx:140]:
```suggestion
To find all memos where content contain ANY of the words `postgres` or `pgroonga`, use the upper case `OR`:
```

File: apps/docs/pages/guides/database/extensions/pgroonga.mdx [hunk 4]
```diff
@@ -100,6 +100,75 @@ The pgroonga index is used to retrive the result set:
...
+```markdown
+ id |                            content                             
+----+----------------------------------------------------------------
+  1 | PostgreSQL is a relational database management system.
+  3 | PGroonga is a PostgreSQL extension that uses Groonga as index.
+(2 rows)
+```
+
+### Search that matches words with negation
+To find all memos where content contain the word postgres but not pgroonga, use - symbol:
```
Comment (dshukertjr) [inline apps/docs/pages/guides/database/extensions/pgroonga.mdx:157]:
```suggestion
To find all memos where content contain the word `postgres` but not `pgroonga`, use `-` symbol:
```

**Review by dshukertjr — COMMENTED** [review]
@askdkc This is fantastic! I noticed a few small things, but otherwise it looks awesome! Thanks for the amazing work!

**Comment by askdkc** [comment]
@dshukertjr 

Thank you for your review! 💕

I apologize for any deficiencies in my written English skills. 😅

File: apps/docs/pages/guides/database/extensions/pgroonga.mdx [hunk 5]
```diff
@@ -100,6 +100,75 @@ The pgroonga index is used to retrive the result set:
 | 2   | 'Groonga is a fast full text search engine that supports all languages.' |
 ```
 
+## Full Text Search
+The `&@~` operator performs full text search. It returns any maching results. Unlike `LIKE` operator, pgroonga can search any text that contains the keyword case insensitive.
```
Comment (dshukertjr) [inline apps/docs/pages/guides/database/extensions/pgroonga.mdx:104]:
```suggestion
The `&@~` operator performs full text search. It returns any matching results. Unlike `LIKE` operator, pgroonga can search any text that contains the keyword case insensitive.
```

**Comment by dshukertjr** [comment]
@askdkc It looks like the prettier check is failing. Would you be able to run prettier on the file?

**Comment by askdkc** [comment]
@dshukertjr 

Sorry about that. Just ran the prettier and made a new commit.

**Review by dshukertjr — APPROVED** [review]
LGTM! It should be merged after a final review from the docs team!

**Comment by askdkc** [comment]
@dshukertjr 

Thank you for reviewing this!

**Review by gregnr — APPROVED** [review]
Looks great, thanks @askdkc!

**Pull request merged**